site/docs/pages/index.mdx

```diff
@@ -25,13 +28,15 @@ import TokenSelector from '../components/TokenSelector';
   <div className="css-blurry-gradient" style={{ top: '35%', left: '10%', height: '220px', width: '220px' }} />
 </div>
 
-<div className="max-w-screen-xl mx-auto vp-doc relative px-[24px] mb-[26px] mt-0 md:px-0 md:mb-[64px]">
-  <div className="mx-auto max-w-2xl">
-    <div className="md:text-center flex flex-col gap-8 pt-48 pb-16">
+<div className="max-w-screen-xl mx-auto vp-doc relative px-[24px] mb-[26px] mt-0 md:px-0 md:mb-[64px]" >
+  <div className="mx-auto max-w-2xl" >
+    <div className="md:text-center flex flex-col gap-8 pt-48 pb-16" >
       <h1 className="text-center text-6xl md:text-9xl font-medium no-underline dark:text-white">OnchainKit</h1>
       <div className="text-center text-xl md:text-3xl dark:text-white">
-        <p className="leading-7 md:leading-none">React components
-        and TypeScript utilities to help you build top-tier onchain apps.</p>
+        <p className="leading-7 md:leading-none">
+          Ship your onchain apps in minutes with out-of-the box React
+          components and TypeScript utilities.
```

Comment:
I honestly don't understand this sentence.

The word `ship` makes me think we are selling a `deploy` tool, which we are not.

Also saying the word `minutes` can really feel miss-leading. And a bit too Marketing vibes, which kind of trows me off as developer.

Reply:
Thoughts on something like - "Build your onchain apps quickly with ready-to-use React components and TypeScript utilities."